src/manage_pg.c

```diff
@@ -1660,7 +1662,7 @@ create_view_vulns ()
          " AS (SELECT DISTINCT nvt FROM results"
          "     WHERE (results.severity != " G_STRINGIFY (SEVERITY_ERROR) "))"
          " SELECT id, uuid, name, creation_time, modification_time,"
-         "        score, qod, 'nvt' AS type"
+         "        score / 10.0 AS severity, qod, 'nvt' AS type"
```

Comment:
Will the sorce / 10.0 stay in these queries or will it be changed in another PR?

Reply:
Yes, it will be replaced again in #1477 which removes the score from NVTs.